content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -70,7 +70,8 @@ different Kubernetes components.
 | `CSIMigrationAzureDisk` | `false` | Alpha | 1.15 | 1.18 |
 | `CSIMigrationAzureDisk` | `false` | Beta | 1.19 | |
 | `CSIMigrationAzureDiskComplete` | `false` | Alpha | 1.17 | |
-| `CSIMigrationAzureFile` | `false` | Alpha | 1.15 | |
+| `CSIMigrationAzureFile` | `false` | Alpha | 1.15 | 1.19 |
+| `CSIMigrationAzureFile` | `false` | Beta | 1.20 | |
 | `CSIMigrationAzureFileComplete` | `false` | Alpha | 1.17 | |
```

Comment:
Oh I forgot about the complete feature gate. Shall we move those to beta too?

Reply:
what does `CSIMigrationAzureFileComplete` really mean? is that just a placeholder flag? I see that flag in other cloud provider is also `Alpha` now?

Reply:
The complete feature gate, when turned on, will cause the intree plugin to not be registered with kube-controller-manager and kubelet. https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-controller-manager/app/plugins_providers.go#L38

Reply:
I think we forgot to move the other drivers feature gates to beta. I plan to clean up the other drivers in 1.21

Reply:
let's clean up that complete flag in all these drivers together in 1.21, thanks.